Misc/NEWS.d/next/Security/2021-01-31-05-28-14.bpo-43075.DoAXqO.rst

```diff
@@ -0,0 +1 @@
+Fix Regular Expression Denial of Service (ReDoS) vulnerability in :class:`urllib.request.AbstractBasicAuthHandler`.  The ReDoS-vulnerable regex has quadratic worst-case complexity and it allows cause a denial of service when identifying crafted invalid RFCs. This ReDoS issue is on the client server and needs remote attackers to control the HTTP server.
```

Comment:
"This ReDoS issue is on the client server"

Do you mean "client side"?

Reply:
Yes, it's the “client side”. I wrote it based on your reviews:joy:, and I'll change it right away. 


> Please explain that the issue is on the client server, the attacker needs to control the HTTP server.